### DISCUSSION TITLE
Image citations for preview images

### DISCUSSION THREAD

**Discussion by DidDrog11** [quarto-dev/quarto-cli#4131] [Q&A]
Is it possible to add a photo credit to a preview image like below? 
https://mmistakes.github.io/minimal-mistakes/layout/uncategorized/layout-header-overlay-image/
Or is it easiest to add one manually to the image itself?

**Comment by dragonstyle** [comment]
Right now I don't know of a way to do this, though it would make a great extension or feature request...
